README.md

```diff
@@ -1,4 +1,143 @@
+
+`SnowFlow_Create_Role.sql` is provided to create this role and assigns the role to a parent role.
+
+You’ll have to update the script before executing it with the information specific to your environment. In the script, you will be providing the parent role. Then you'll use the parent role when logging in.
+
+### Step 3 – Install Excel Add-in
+
+In order to install the plugin, which Excel calls "add-ins," make sure you have access to the "Developer" tab within Excel's menu.  You can add it by right-clicking on the toolbar, selecting “customize the ribbon", and then clicking its checkbox.
+
+![](images/image1.png)
+
+Now select Excel Add-Ins:
+
+![](images/image2.png)
+
+Navigate to the directory where the `.xlam` file was saved.  Make sure to check the plugin before selecting OK.
+
+<img src="images/image3.png" width=50% >
+
+At this point, Excelerator is available within Excel. You'll see a UI within the "Home" tab.
+
+## Use Excelerator
+
+With Excelerator installed, now you need to connect it to Snowflake. 
+
+![](images/image4.png)
+
+First, confirm you can find the appropriate buttons within the "Home" tab.
+
+### Connection Parameters
+
+Click the `connect` button and enter your Snowflake connection information.  There are two security types standard: login/password and SSO.  We recommend using SSO for more secure authentication.
+
+![](images/image5.png)
+
+In the `Role` field, add the role that you entered into the script in Step 3.
+
+| :exclamation:        | The login database and schema specified in the connection must be the same database and schema the Stored Procs were run under     |
+|---------------|:------------------------|
+
+### Execute a Query
+
+Now we'll walk you through each section of the plugin. You can work with data sourced from other data sources within Excel (such as files) or you can pull data from Snowflake into Excel.  You can manipulate the data using VBA scripts and vlookups, but be sure to write that final data is sent back to Snowflake! The plugin handles both reading and writing scenarios.
+
+Pull data into Excel by selecting the “Query button” and typing in the query.
+
+![](images/image6.png)
+
+You'll get results that look similar to this:
+
+![](images/image7.png)
+
+Repeat as-needed to gather data from Snowflake into your Excel sheet.
+
+### Write Data to Snowflake
+
+Once the data is returned you can update or insert rows and columns within Excel.
+
+To propagate those changes to Snowflake press the “Upload” button.  Here you can enter the worksheet the data is being sourced from and define the table the data is being written to.
+
+Another important option is defining the columns that will represent the keys in order to do the update logic needed if the user updates or adds a new column in the Excel sheet.  This can be done by entering the column position ex. A,B,C.  So the first 3 columns in the excel sheet and target table will be the key
```

Comment:
Similar, we aren't sure what's being communicated here, but don't want to edit out important information!